pkg/compose/stop.go

```diff
@@ -54,6 +54,7 @@ func (s *composeService) stop(ctx context.Context, projectName string, options a
 		if !utils.StringContains(options.Services, service) {
 			return nil
 		}
-		return s.stopContainers(ctx, w, containers.filter(isService(service)).filter(isNotOneOff), options.Timeout)
+		serv := project.Services[service]
```

Comment:
(repeat)
We could add a validation that the service exists here. Since this is the place where we get its value becoming easier later on for debugging.

serv, ok := project.Services[service] 
if !ok {
  return fmt.Error() ....
}